docs/installation/installation.md

```diff
@@ -185,9 +185,15 @@ kubectl create secret -n tyk-operator-system generic tyk-operator-conf \
 If you don't have cert-manager installed, here is a quick install:
 
 ```bash
-kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.0.3/cert-manager.yaml
+kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.3.0/cert-manager.yaml
 ```
 
+This cert-manager version is the minimum version that works with kubernetes version 1.22 . 
+For kubernetes versions below 1.22, earlier releases of cert-manager  should work. 
```

Comment:
We may want to forward users to https://cert-manager.io/docs/installation/supported-releases/ page to check which earlier releases should work with their specific Kubernetes version.